src/main/java/com/flightstats/hub/dao/aws/S3SingleContentDao.java

```diff
@@ -327,6 +327,7 @@ public static ObjectMetadata createObjectMetadata(Content content, boolean useEn
         } else {
             metadata.addUserMetadata("type", "none");
         }
+        metadata.addUserMetadata("payload", "none");
```

Comment:
It sets the payload metadata to none obviously :D Honestly I'm not sure what the intent was there. I'm going to double check that its not a loose end that needs tied up and just remove it entirely if so.